src/javascripts/jquery.tipsy.js

```diff
@@ -118,6 +118,21 @@
                 }
             }
         },
+
+        destroy: function(){
+          $.removeData(this.$element.get(0));
```

Comment:
Ahh ok, thought you meant the argument passed to it. Maybe @oscarwallhult has some insight?